multi-educator/src/Dscp/Educator/Launcher/Entry.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE OverloadedLabels #-}
+
+-- | Educator entry point.
+
+module Dscp.Educator.Launcher.Entry
```

Comment:
All your modules should not repeat `Dscp.Educator` modules, they should be called something like `Dscp.MultiEducator` (or `Dscp.Educator.Multi`, whatever) and _import_ almost everything from `Dscp.Educator` modules.

Reply:
This is precisely what I was going to do.